tools/kompos/src/debugger.ts

```diff
@@ -8,6 +8,14 @@ export function isRelevant(sc: TaggedSourceCoordinate) {
   return -1 !== sc.tags.indexOf("ExpressionBreakpoint");
 }
 
+export function isRelevantPair(sc: TaggedSourceCoordinate) { 
+  return -1 !== sc.tags.indexOf('CreatePromisePair');
```

Comment:
Perhaps you could just include the `ExpressionBreakpoint` tag on those nodes as well. It seems strange to have extra tests here, while the other breakpoints unified this nicely.